src/shared/Microsoft.Git.CredentialManager/Interop/Linux/LinuxCredentialStore.cs

```diff
@@ -11,29 +11,33 @@ public class LinuxCredentialStore : ICredentialStore
     {
         private const string SecretServiceStoreOption = "secretservice";
         private const string GpgStoreOption = "gpg";
+        private const string CredentialCacheOption = "credentialcache";
```

Comment:
I would say we should make this value be either the kebab-case form `credential-cache`, or simply just `cache`.

This leaves the possibility for us to allow arbitrary `git-<cmd>` or `git-credential-<cmd>` command forwarding at some point.. what do you think?

Reply:
No opinion personally. `cache` is fine by me.